docs/understand/sizing-guide.md

```diff
@@ -296,9 +283,75 @@ So, 50% of all requests took <15ms, while 99.9% of them took <100ms.
 
 The average throughput during the experiment was **7069.03 requests/second**.
 </div>
+<div markdown="1" id="dynamodb-ram">
+All benchmarks below were measured using m5.xlarge instance on AWS us-east-1.
```

Comment:
Please add more information for both the lakeFS container used and the repository data used for the benchmark - same as is Postgres

Reply:
done